v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/transforms/AvroTransform.java

```diff
@@ -185,6 +191,9 @@ public void processElement(ProcessContext context) {
               element.getPayload(),
               BigQueryUtils.toTableSchema(
                   AvroUtils.toBeamSchema(element.getPayload().getSchema())));
+      if (this.useKey) {
+        row.set("_key", element.getOriginalPayload().getKV().getKey());
```

Comment:
`_key` - I would make this a static variable at the top of the class.

Reply:
Done